ApplicationCode/GeoMech/GeoMechDataModel/RigFemPartResultCalculatorStressAnisotropy.cpp

```diff
@@ -0,0 +1,140 @@
+//
+//  Copyright (C) 2020-     Equinor ASA
+//
+//  ResInsight is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  ResInsight is distributed in the hope that it will be useful, but WITHOUT ANY
+//  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+//  FITNESS FOR A PARTICULAR PURPOSE.
+//
+//  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+//  for more details.
+//
+/////////////////////////////////////////////////////////////////////////////////
+
+#include "RigFemPartResultCalculatorStressAnisotropy.h"
+
+#include "RigFemPart.h"
+#include "RigFemPartCollection.h"
+#include "RigFemPartResultsCollection.h"
+#include "RigFemResultAddress.h"
+#include "RigFemScalarResultFrames.h"
+
+#include "cafProgressInfo.h"
+
+#include <QString>
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+RigFemPartResultCalculatorStressAnisotropy::RigFemPartResultCalculatorStressAnisotropy( RigFemPartResultsCollection& collection )
+    : RigFemPartResultCalculator( collection )
+{
+}
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+RigFemPartResultCalculatorStressAnisotropy::~RigFemPartResultCalculatorStressAnisotropy()
+{
+}
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+bool RigFemPartResultCalculatorStressAnisotropy::isMatching( const RigFemResultAddress& resVarAddr ) const
+{
+    return (
+        ( ( resVarAddr.fieldName == "ST" ) && ( resVarAddr.componentName == "STA12" || resVarAddr.componentName == "STA13" ||
+                                                resVarAddr.componentName == "STA23" ) ) ||
+        ( ( resVarAddr.fieldName == "SE" ) && ( resVarAddr.componentName == "SEA12" || resVarAddr.componentName == "SEA13" ||
+                                                resVarAddr.componentName == "SEA23" ) ) );
+}
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+RigFemScalarResultFrames* RigFemPartResultCalculatorStressAnisotropy::calculate( int                        partIndex,
+                                                                                 const RigFemResultAddress& resVarAddr )
+{
+    CVF_ASSERT( isMatchine( resVarAddr ) );
```

Comment:
Typo ?

Reply:
Yes, I guess it does not fail in release mode. I will fix.

Reply:
Fixed now.